packages/mobile/src/recipients/recipient.ts

```diff
@@ -140,19 +124,29 @@ export function getRecipientFromAddress(
   return e164PhoneNumber ? recipientCache[e164PhoneNumber] : undefined
 }
 
-export function getRecipientVerificationStatus(
+export const getRecipientVerificationStatus = (
```

Comment:
why change this?

Reply:
Part of a larger refactor...basically folded 3 functions across 3 different files into 1. Think it's a net positive. Will change back to non-arrow function as per earlier style comment.